.github/dependabot.yml

```diff
@@ -6,6 +6,9 @@ updates:
       interval: "daily"
     allow:
       - dependency-type: "all"
+    ignore:
+      # Ginkgo update needs a change in automation/build-test.sh.
+      - dependency-name: "github.com/onsi/ginkgo/v2"
```

Comment:
If we hardcode the gomega version in `automation/build-test.sh` it should be ignored here too.

Reply:
fixed